dss-api.yml

```diff
@@ -1224,6 +1224,18 @@ paths:
       description: >
         Given a bundle UUID, return the latest version of that bundle.  If the version is provided, that version of the
         bundle is returned instead.
+
+        # Pagination
+
+        The DSS API supports pagination in a manner consistent with the
+        [GitHub API](https://developer.github.com/v3/guides/traversing-with-pagination/),
+        which is based on [RFC 5988](https://tools.ietf.org/html/rfc5988). When the results of an API call exceed the
+        page size specified, the HTTP response will contain a `Link` header of the following form:
+        `Link: <https://dss.data.humancellatlas.org/v1/search?replica=aws&per_page=100&start_at=123>; rel="next"`.
```

Comment:
Should this be documented in the response spec?

Reply:
Moved.